src/main/java/org/testng/annotations/Parameters.java

```diff
@@ -14,7 +13,7 @@
  * @author <a href="mailto:cedric&#64;beust.com">Cedric Beust</a>
  */
 @Retention(RUNTIME)
-@Target({METHOD, CONSTRUCTOR, TYPE })
+@Target({METHOD, TYPE})
 public @interface Parameters {
```

Comment:
I don't understand why this is being removed either, or even deprecated. It's useful and there is no other way to implement this functionality.